airbyte-integrations/connectors/source-faker/source_faker/spec.json

```diff
@@ -35,8 +35,16 @@
         "description": "How many fake records will be in each page (stream slice), before a state message is emitted?",
         "type": "integer",
         "minimum": 1,
-        "default": 100,
+        "default": 1000,
         "order": 3
+      },
+      "threads": {
+        "title": "Worker Threads",
```

Comment:
can we call this `Number of parallel data generation processes` or something that: 
1) doesn't use the word `Worker` (my guess is the user doesn't know what a worker is) 
2) doesn't contain the word `threads`